api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -275,8 +275,8 @@
          *     <li>A nested LRA which has closed must retain the ability to cancel the
          *     effects if the the parent cancels. This requirement must be enforced
          *     by participants.
-         *     <li>If a nested LRA cancels then all of its children must cancel (even if
-         *     they closed - see 2).
+         *     <li>If a nested LRA cancels then all of its children must cancel (or compensate
+         *     if they are already closed - see 2).
```

Comment:
This is a little confusing to me. Both LRAs and participants need to cancel/compensate even if they previously closed/completed. We shouldn't mix the terminology (LRA -- close/cancel, participant -- complete/compensate).

Reply:
And it is not the nested LRA itself, but the parent of the nested LRAs that cancels


`if an LRA cancels then all the participants linked to the nested LRA children of this parent LRA need to compensate, even if they are already completed - see 2)`

But probably the word 'linked' is not the best term.

Reply:
I have added more clarifying text.